### PR TITLE
Meeting spot notification

### PR DESCRIPTION
**Pull request by mtv47**
Now if you long click on a stop in the Map, a meeting notification is send to all users present in a Trip

Also fixed a bug where already finished suggestions were still being shown on the Map and Stops where hidden because of this.

Added tests for the Notification part of the code as well

### CODE REVIEW

**Review by tosigame — CHANGES_REQUESTED** [review]
Looks good, clean way to add an announcement. However the code coverage is low, and you should test the NotificationManager method you added. (Its easy, you just have to mockk the repository and you can see how its done in the viewmodel tests). Hope it helps!, good luck!

**Comment by mtv47** [comment]
> Looks good, clean way to add an announcement. However the code coverage is low, and you should test the NotificationManager method you added. (Its easy, you just have to mockk the repository and you can see how its done in the viewmodel tests). Hope it helps!, good luck!

I've added a NotificationsManagerTest to test all methods but unfortunately I wasn't able to mock LocalDateTime.now()

File: app/src/test/java/com/github/se/wanderpals/service/NotificationsManagerTest.kt
Comment (tosigame) [inline app/src/test/java/com/github/se/wanderpals/service/NotificationsManagerTest.kt:1]: Great, you tested everything in the new function. And even tested the whole NotificationManager, while at it. Great job!

**Review by tosigame — APPROVED** [review]
Great job, works fine for me , and good job on testing the whole class, and thank you for your responsiveness.  As always excellent Job 🎖️

**Pull request merged**